civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.features.menu_links.inc

```diff
@@ -28,17 +28,26 @@ function civihr_employee_portal_features_menu_default_menu_links() {
     'weight' => 0,
```

Comment:
Is there any reason changes were done to only certain files? 

`ls -lR | awk '/^-/ { print $1 " " $9 }' | sort`

Shows that there are still a mix of file permissions

Reply:
Where did you run that command? Here is what I get on my local:

```
 /projetos/civihr/hr17/sites/all/modules/civihr-custom/civihr_employee_portal/features/civihr_employee_portal_features   PCHR-3037-report-listing-changes  ls -lR | awk '/^-/ { print $1 " " $9 }' | sort
-rw-rw-r-- civihr_employee_portal_features.features.fe_block_settings.inc
-rw-rw-r-- civihr_employee_portal_features.features.field_base.inc
-rw-rw-r-- civihr_employee_portal_features.features.field_instance.inc
-rw-rw-r-- civihr_employee_portal_features.features.inc
-rw-rw-r-- civihr_employee_portal_features.features.menu_custom.inc
-rw-rw-r-- civihr_employee_portal_features.features.menu_links.inc
-rw-rw-r-- civihr_employee_portal_features.features.taxonomy.inc
-rw-rw-r-- civihr_employee_portal_features.info
-rw-rw-r-- civihr_employee_portal_features.install
-rw-rw-r-- civihr_employee_portal_features.module
-rw-rw-r-- civihr_employee_portal_features.pages_default.inc
-rw-rw-r-- civihr_employee_portal_features.rules_defaults.inc
-rw-rw-r-- civihr_employee_portal_features.strongarm.inc
-rw-rw-r-- civihr_employee_portal_features.views_default.inc
 /projetos/civihr/hr17/sites/all/modules/civihr-custom/civihr_employee_portal/features/civihr_employee_portal_features   PCHR-3037-report-listing-changes  git checkout staging                         
Switched to branch 'staging'
Your branch is up-to-date with 'origin/staging'.
 /projetos/civihr/hr17/sites/all/modules/civihr-custom/civihr_employee_portal/features/civihr_employee_portal_features   staging  ls -lR | awk '/^-/ { print $1 " " $9 }' | sort
-rw-rw-r-- civihr_employee_portal_features.features.fe_block_settings.inc
-rw-rw-r-- civihr_employee_portal_features.pages_default.inc
-rw-rw-r-- civihr_employee_portal_features.views_default.inc
-rwxrwxr-x civihr_employee_portal_features.features.field_base.inc
-rwxrwxr-x civihr_employee_portal_features.features.field_instance.inc
-rwxrwxr-x civihr_employee_portal_features.features.inc
-rwxrwxr-x civihr_employee_portal_features.features.menu_custom.inc
-rwxrwxr-x civihr_employee_portal_features.features.menu_links.inc
-rwxrwxr-x civihr_employee_portal_features.features.taxonomy.inc
-rwxrwxr-x civihr_employee_portal_features.info
-rwxrwxr-x civihr_employee_portal_features.install
-rwxrwxr-x civihr_employee_portal_features.module
-rwxrwxr-x civihr_employee_portal_features.rules_defaults.inc
-rwxrwxr-x civihr_employee_portal_features.strongarm.inc
```

As you can see, on the PR branch, all the permissions of all files are set to 644, whereas on staging some have 644 and others 755.

Reply:
I meant in the civihr_employee_portal directory, but they're outside of the scope of this PR so approving